common/ringbuffer.c

```diff
@@ -58,8 +58,6 @@ LIB_EXPORT void jack_ringbuffer_reset(jack_ringbuffer_t *rb);
 LIB_EXPORT void jack_ringbuffer_reset_size (jack_ringbuffer_t * rb, size_t sz);
 LIB_EXPORT size_t jack_ringbuffer_write(jack_ringbuffer_t *rb, const char *src,
                                  size_t cnt);
-void jack_ringbuffer_write_advance(jack_ringbuffer_t *rb, size_t cnt);
```

Comment:
This seems radical.

Reply:
https://github.com/jackaudio/jack2/blob/98c882c5a545c3dd819655c9869f6928274df8a3/common/jack/ringbuffer.h#L228
It's defined twice, why define the prototype in a c-file when it's defined in the header? But like I said, I only have basic c++ knowledge, so this could be more radical than I realized